test/e2e/plain_provisioner_test.go

```diff
@@ -351,115 +351,100 @@ var _ = Describe("plain provisioner bundle", func() {
 
 	When("Bundles are backed by a git repository", func() {
```

Comment:
For some reason, I can't target this test during a focus as its written now. I'm not sure whether its due to the way it's written but if it's not running during the overall e2e run then I will have to change it. It may not be finding the spec inside.

Reply:
Yep I didn't see that When get run in the latest e2e-kind action output. Super weird.